src/commons/navbar-searchbar/navbar/navbar.js

```diff
@@ -51,7 +117,11 @@ export class Navbar {
   }
 
   navigateTo(path) {
-    this.router.navigate(path)
+    this.router.navigate(path);
   }
 
+  detached() {
+    this.routerSubscription.dispose();
+    this.subscription.dispose();
```

Comment:
Are you sure that this.subscription.unsubscribe() and  this.subscription.dispose() do the same?
Official documentation suggests to use this.subscription.unsubscribe();